packages/fluentui/react-bindings/src/hooks/useFrameFocusDispatch.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+export const useFrameFocusDispatch = (enableFrameFocusDispatch: boolean, $document: Document) => {
+  const timeoutRef = React.useRef<ReturnType<typeof setTimeout>>();
+
+  React.useEffect(() => {
+    if (enableFrameFocusDispatch) {
+      timeoutRef.current = setInterval(() => {
+        if ($document.activeElement?.tagName === 'IFRAME') {
+          const event = new CustomEvent(FUI_FRAME_EVENT, { bubbles: true });
+          $document.activeElement.dispatchEvent(event);
+        }
+      }, 1000);
```

Comment:
should the length of the interval be configurable ?

Reply:
it would tie in with your proposal to hoist some configuration to the provider level.